js/mojs-player.babel.js

```diff
@@ -400,6 +400,12 @@ class MojsPlayer extends Module {
   */
   _onHideStateChange ( isHidden ) {
     this._props.isHidden = isHidden;
+    const { onToggleHide } = this._o;
+    // skip running callback on init rendering
+    if ( this._hideCount && onToggleHide) {
```

Comment:
To tell the truth I thought that it shouldn't fire callback on initial rendering (because of this line in issue description 'This callback should be called after user taps on the button'), so that's why I've added this 🙃